.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: GitHub App token
+        id: github_app_token
+        uses: tibdex/github-app-token@v1.5.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          installation_id: 22958780
+      - name: Get tag
+        id: tag
+        uses: dawidd6/action-get-tag@v1
```

Comment:
This project `dawidd6/action-get-tag` is archived, also it seems it's not used by this workflow.

Reply:
Ya @ruanyl, since I'm using this default `${GITHUB_REF/refs\/tags\//}` I can remove this extra step.